targets/TARGET_STM/TARGET_STM32L0/serial_device.c

```diff
@@ -79,17 +79,15 @@ static void uart2_irq(void)
 }
 #endif
 
-#if defined(USART4_BASE)
-static void uart4_irq(void)
+#if defined(USART4_BASE) || defined(USART5_BASE)
+static void uart4_5_irq(void)
 {
+    #if defined(USART4_BASE)
     uart_irq(UART_4);
-}
-#endif
-
-#if defined(USART5_BASE)
-static void uart5_irq(void)
-{
+    #endif
+    #if defined(USART4_BASE)
```

Comment:
Preprocessor macros always at the line beginning, as it was in the file before, please align

Reply:
Just pushed this.